tests/foreman/api/test_reporttemplates.py

```diff
@@ -378,6 +377,10 @@ def test_negative_create_report_without_name():
 
     :CaseImportance: Medium
     """
+    template1 = gen_string('alpha')
+    with pytest.raises(HTTPError) as report_response:
+        module_target_sat.api.ReportTemplate(name=' ', template=template1).create()
```

Comment:
```suggestion
    with pytest.raises(HTTPError) as report_response:
        module_target_sat.api.ReportTemplate(name=' ', template=gen_string('alpha')).create()
```

Reply:
Thank you for your suggestion. changes accepted.